scenarios/reproducers/3-nodes.yml

```diff
@@ -0,0 +1,66 @@
+---
+cifmw_install_yamls_repo: /home/cjeanner/work/github/openstack-k8s-operators/install_yamls
+cifmw_basedir: "{{ ansible_user_dir }}/ci-framework-data"
+cifmw_path: "{{ ansible_user_dir }}/.crc/bin:{{ ansible_user_dir }}/.crc/bin/oc:{{ ansible_user_dir }}/bin:{{ ansible_env.PATH }}"
+cifmw_reproducer_repositories:
+  - src: "{{ playbook_dir }}"
+    dest: "src/github.com/openstack-k8s-operators/"
+  - src: "{{ cifmw_install_yamls_repo }}"
+    dest: "src/github.com/openstack-k8s-operators/"
+cifmw_libvirt_manager_configuration:
+  vms:
+    compute:
+      amount: 1
+      image_url: "{{ cifmw_discovered_image_url }}"
+      sha256_image_name: "{{ cifmw_discovered_hash }}"
+      image_local_dir: "{{ cifmw_basedir }}/images/"
+      disk_file_name: "centos-stream-9.qcow2"
+      disksize: 50
+      memory: 8
+      cpus: 4
+      nets:
+        - default
+        - crc
+    controller:
+      image_url: "{{ cifmw_discovered_image_url }}"
+      sha256_image_name: "{{ cifmw_discovered_hash }}"
+      image_local_dir: "{{ cifmw_basedir }}/images/"
+      disk_file_name: "centos-stream-9.qcow2"
+      disksize: 50
+      memory: 8
+      cpus: 4
+      nets:
+        - default
+        - crc
+    crc:
+      admin_user: core
+      image_local_dir: "{{ ansible_user_dir }}/.crc/machines/crc/"
+      disk_file_name: "crc.qcow2"
+      disksize: 150
+      memory: 32
+      cpus: 24
+      nets:
+        - default
+        - crc
+  networks:
+    default: |-
+      <network>
+        <name>default</name>
+        <forward mode='nat'/>
+        <bridge name='virbr0' stp='on' delay='0'/>
+        <mac address='52:54:00:6a:f2:dc'/>
+        <ip family='ipv4' address='192.168.100.1' prefix='24'>
+          <dhcp>
+            <range start='192.168.100.10' end='192.168.100.100'/>
```

Comment:
it is possible to make reservation for each machine for each mac.

Reply:
"Grab IPs for nodes type"  is not needed if you know the address in advanced.

Reply:
maybe. But we'd still need to wait for ssh to come up. I'd rather avoid going with static IPs in there, to be fair.